wal/metrics.go

```diff
@@ -17,6 +17,13 @@ package wal
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
+	syncGauge = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "etcd",
+		Subsystem: "disk",
+		Name:      "wal_fsync_duration_current",
```

Comment:
I actually jumped here from the other issue and only realized after there's a long discussion around this. As per my comment in the issue, I think the last observation on its own is not the best option for capturing over-all latency of the WAL.